packages/rspack/src/compiler.ts

```diff
@@ -353,7 +359,7 @@ class Compiler {
 					buildModule: this.#buildModule.bind(this)
 				},
 				createThreadsafeNodeFSFromRaw(this.outputFileSystem),
-				loaderContext => runLoader(loaderContext, this)
+				(loaderContext: any) => runLoader(loaderContext, this)
```

Comment:
What is this for?

Reply:
> What is this for?

In my editor, an error will be reported  : 参数“loaderContext”隐式具有“any”类型。
Maybe it can be changed to this?

```js
(loaderContext: binding.JsLoaderContext) => runLoader(loaderContext, this)
```

Reply:
Could you try `import type * as binding from "@rspack/binding";`? Since NAPI-RS generates named exports in `d.ts` file.

Reply:
I tried it, still getting an error. So I changed it to this
```js
(loaderContext: binding.JsLoaderContext) => runLoader(loaderContext, this)
```